shared/constants/network.ts

```diff
@@ -343,6 +347,19 @@ export const NETWORK_TO_NAME_MAP = {
   [CHAIN_IDS.LOCALHOST]: LOCALHOST_DISPLAY_NAME,
 } as const;
 
+export const CHAIN_ID_TO_CURRENCY_SYMBOL_MAP = {
```

Comment:
Where is this used?

Reply:
The usage isn't fully refined/discussed yet, but I think the future intention is that it should be used as a shortlist to query against so we can avoid calling chainlist API for 90% of our users who have that setting turned on